cli/cmd/generate_azure.go

```diff
@@ -738,6 +773,11 @@ func promptAzureGenerate(config *azure.GenerateAzureTfConfigurationArgs, extraSt
 		return errors.New("must enable activity log or config")
 	}
 
+	//GROW-1444: Prompt for Azure SubscriptionID
```

Comment:
What is the benefit of having this comment?  Git logs/annotate/blame should link to context and link to Jira.

I don't see this as providing any additional information - function name `askAzureSubscriptionID` says it all.